docs/source/configuration.mdx

```diff
@@ -78,6 +78,10 @@ server:
       - http://localhost
 ```
 
+## Logging
+
+Apollo Router detects if the stdout is a TTY and enable JSON logging only if it is not.
```

Comment:
```suggestion
Apollo Router detects if the stdout is a TTY and enables JSON logging only if it is not.
```

Not english native, please discard if it doesn't make any sense 😅

Reply:
:+1: